src/widgets/conditions/conditions-detailed-summary.css

```diff
@@ -0,0 +1,51 @@
+:global(.omrs-breakpoint-gt-phone) .conditionSummary {
+  display: flex;
+}
+
+:global(.omrs-breakpoint-lt-tablet) .conditionSummary {
+  padding: 0.25rem 0.5rem 0 0.5rem;
+}
+
+.conditionCard {
+  margin: 0rem 1rem;
+  font-size: 1.063rem;
```

Comment:
Why such an unusual font size?

Reply:
17px in keeping with the design, though 16px (which neatly converts to 1rem) would probably be just as appropriate (and makes for a cleaner conversion). Should I go with 17px as per the design?

Reply:
Please stick with design.

Reply:
Ok. 1.063 rem it is.